lib/postprocessing/image.js

```diff
@@ -227,6 +227,11 @@ async function applyOutputProperties(img, instructions) {
         img.define(`jpeg:extent=${instructions.jpegSize}`);
     }
 
+    // make sure colorspace is sRGB: bug with CMYK files producing CMYK images
+    if (instructions.fmt === "jpg" || instructions.fmt === "jpeg" || instructions.fmt === "png") {
+        img.colorspace('sRGB');
```

Comment:
As discussed, we should always set it regardless of output format (unless imagemagick doesn't like it for other file types?).

Reply:
What does the function actually guarantee? JPEG, PNG, GIF output compatible with the OOTB workers? If so, it should also define that the result is always RGB.

Reply:
Comment explanation would be something like: "Most asset compute renditions are used in digital display context (web, screens), where RGB is required"

Reply:
I changed it to set `sRGB` for all file formats. I also added a cmykJpg -> gif test to verify this works for gif formats.

Reply:
btw, any preference between `sRGB` and `RGB`?